src/MSCorEE.Desktop/MSCorEE.cs

```diff
@@ -14,6 +16,11 @@ namespace PInvoke
     public static partial class MSCorEE
     {
         /// <summary>
+        /// The CLSID that may be passed to <see cref="CLRCreateInstance"/> to create an instance of <see cref="ICLRMetaHost"/>.
+        /// </summary>
+        public static readonly Guid CLSID_CLRMetaHost = new Guid("{9280188d-0e8e-4867-b30c-7fa83884e8de}");
```

Comment:
Found it:
https://msdn.microsoft.com/en-us/library/dd537633(v=vs.110).aspx
and in the metahost.h file.